module_auto_update/tests/test_module.py

```diff
@@ -79,8 +79,7 @@ def test_get_modules_with_changed_checksum(self):
         self.assertFalse(Imm._get_modules_with_changed_checksum())
 
 
-@common.at_install(False)
-@common.post_install(True)
+@odoo.tests.tagged('post_install', '-at_install')
```

Comment:
Is this fix on tests generic? I mean, is it required in general for every other module? In that case, it would be good to add this fix in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0

Reply:
I've not investigated in depth, but at first glance yes, it looks like an Odoo backward compatibility bug.

Reply:
Reported to Odoo: https://github.com/odoo/odoo/issues/27471

Reply:
Thank you!